.travis.yml

```diff
@@ -19,5 +20,9 @@ matrix:
       env: TOXENV=flake8
 install:
   - pip install tox-travis
+  - npm install -g grunt-cli
+  - npm install grunt
+  - npm install grunt-contrib-qunit
 script:
   - tox
+  - grunt test
```

Comment:
I think we've talked about that in Mountain View: we thought that running 16 times would not be a big impact to the runtime, and it would be weird if one pytest fails to js and other no and failing them all would be more intuitive (it is not very intuitive, at the end :P)... But I liked the idea of setting up CircleCI to run JS tests, have you ever done a project with both CI at the same time? How does it works?